### PR TITLE
{data}[foss/2016b] ITK v4.12.2

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 8.9, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.9
See https://gist.github.com/6469484feae4a26bc7de6d066a53a49a for a full test report.

**Comment by vanzod** [comment]
The build takes quite some time and Travis is quite impatient. Everything else passes though.

File: easybuild/easyconfigs/i/ITK/ITK-4.12.2-foss-2016b-Python-2.7.12.eb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+preinstallopts = "export PYTHONPATH=%(installdir)s/lib/python%(pyshortver)s/site-packages:$PYTHONPATH && "
+
+separate_build_dir = True
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
+
+sanity_check_paths = {
+    'files': ['bin/itkTestDriver',
+              'lib/libITKVtkGlue-4.12.a', 'lib/libitkhdf5.a',
+              'lib/libitkjpeg-4.12.a', 'lib/libitkpng-4.12.a'],
```
Comment (boegel) [inline easybuild/easyconfigs/i/ITK/ITK-4.12.2-foss-2016b-Python-2.7.12.eb]: @vanzod please use `%(version_major_minor)s` rather than hardcoding `4.12`?

**Comment by boegel** [comment]
@vanzod Travis doesn't do the actual build, it just runs the unit tests... I re-triggered the test that timed out, usually the tests finish just fine in that time window.

File: easybuild/easyconfigs/i/ITK/ITK-4.12.2-foss-2016b-Python-2.7.12.eb (hunk 1 above)
Comment (vanzod) [reply easybuild/easyconfigs/i/ITK/ITK-4.12.2-foss-2016b-Python-2.7.12.eb]: Good catch

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2027.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1c8f3b0efad6818a5986a4bf3fbb9383 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (1 easyconfigs in this PR)
node2641.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/763c8a02ac59509108b6ce93424a834c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**